content/ja/handbook/marketing/developer-relations/technical-marketing/howto/publish_video_on_youtube.md

```
@@ -0,0 +1,142 @@
+---
+title: "YouTube に動画を公開する"
+upstream_path: /handbook/marketing/developer-relations/technical-marketing/howto/publish_video_on_youtube/
+upstream_sha: 12455b65f7f70cf1c18488a44bbab3cff0097c8a
+translated_at: "2026-05-01T00:00:00Z"
+translator: claude
+stale: false
+---
+
+### 1. 動画をアップロードする
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)